### DISCUSSION TITLE
Negative feedback - first social network implementing this? Anonymous likeing -5 .. +5?

### DISCUSSION THREAD

**Discussion by mig1984** [mastodon/mastodon#16359] [General]
When you give a like or a star, it's always positive feedback. It causes amplification. That's why it happens that a complete banality becomes a trend on social media.

I would like to see a better balanced social network where both positive and negative feedback is possible. 

Moreover, a like or a star alone does not mean relevance. I don't know "how much" the person in question liked or disliked my post. Nowadays people do like automatically, so it doesn't mean anything without that "how much" information.

I would suggest adding an optional "thermometer" with -5 .. +5 range. In contrast to the star, it would be fully anonymous, so user will see a total but not who voted how (otherwise people wouldn't vote negative -> it wouldn't be relevant and therefore useless).

**Comment by weex** [comment]
One thing I really like on slashdot is moderation going both ways + and -. A BS button would be awesome to see, wherein I could be see some kind of tag on a toot if one of my close friends has flagged it or what it shares as deceptive or literally BS.

I'd encourage you to refine your idea toward the problem you're trying to solve, leaving solutions for the implementer and post as an issue.

**Reply by Zarkon55555** [reply]
For Mastodon to make a difference I think it needs several "improvements"

I'm looking into forking Mastodon to include the following:

Community Policing - a way for the community as a whole (by instance I think) to up/down vote a member's Reputation and InternetAge scores - these would be useful in filtering posts to "punish" bad actors (reducing visibility without truly censoring them) - users would have the ability set a minimum Reputation and/or InternetAge score in their filter setting for visibility in their feeds

Better filters - needed (AND/OR/NOT) combo filters and the ability to filter on Reputation and InternetAge scores.

InternetAge would be earned only by TIME/AGE of account plus initial "meat-space" verification.
(Example new user's InternetAge = 0 until meat-space verified then = 1 and increments with time from there on)

Exclusion of non-humans (no Bots, Corporations etc.) each user account must equal one human in meat-space - must prove to another verified meat-space user that they are a real human person - this does NOT mean ID verification - just that you're a real person

Persons that validate non-humans could be penalized by negative InternetAge from the admin if discovered

A user's InternetAge points would be "spent" to up/down vote the Reputation score of someone they like/dislike, at a quadratic cost of 4 InternetAge points for each point of change in the Reputation score you are trying to modify

The InternetAge score would increment gradually over time (TBD interval - example +1 per month)
(Basically a user account that has never used any InternetAge points to mod another user would just be equivalent to the AGE of the account)

The end goal is an instance-based community able to self-police their users

Posts from bad actors would slowly fade/disappear in feeds based on default filter settings (changeable by each user). Users with InternetAge  <= 0 (unverified in meat space) would have no vote (to help prevent bots)

Reputation and InternetAge would be based on user-by-user basis not on a post-by-post basis

To "Federate" this idea - Instances would have their own InternetAge/Reputation scores

Reputation & InternetAge would be hidden on posts and profiles to prevent users from trying to game the system or create popularity contests

Users can see their OWN scores in settings/profile, but NOT who up/down voted them to prevent retaliation

**Reply by weex** [reply]
Moderation is a hard problem and I think a fork to try out your strategy could be very interesting. As maintainer of the Ecko fork I recognize now that there's a fair amount of work and responsibility that comes after the press of that little button. To the extent that you are addressing a valid problem we'd probably welcome your contributions.

**Comment by kashfshah** [comment]
Hacker News has a fairly well thought out approach to providing negative feedback, but even that has some downsides. I can go into detail about that, if there is interest in the topic, but what I would like to mention is the following:

Negative feedback can already be provided though the moderation system via reports (negative feedback may happen through moderation action) or through the creation of polls via specific questions (negative feedback may happen through yes/no, agree/disagree questions, etc.).

Most people appreciate positive feedback, but negative feedback can often be misused to induce psychological harm.

Any thoughts on other ways that negative feedback could be provided in a psychologically safe manner?

**Reply by kashfshah** [reply]
Losing a little sleep thinking about this topic and recognized that psychosocial harm and psychosocial safety are the more correct terms.
